scriptworker/constants.py

```diff
@@ -354,9 +360,12 @@
                 ),
             }),
             'mobile': frozendict({
-                'focus-repo': (
+                'focus-origin-repo': (
                     '/mozilla-mobile/focus-android',
                 ),
+                'android-components-origin-repo': (
+                    '/mozilla-mobile/android-components',
+                ),
```

Comment:
Should we prep things in advance by adding the`reference-browser` counterpart too, given upcoming work?

Reply:
I'd prefer @mitchhentges to handle this change. This looks like a good first patch in scriptworker, to me.

Reply:
Good point.

Reply:
By the way, that makes me wonder if shouldn't remove https://github.com/mozilla-releng/scriptworker/pull/271/files#diff-4f43b126671fd39d1f9e922fbb72b839R214 too. That may be confusing during the development of `reference-browser`.

Reply:
In the interest of good-first-patch, that makes sense.

Reply:
Done

Reply:
https://github.com/mozilla-releng/scriptworker/commit/f45ea14a383b55987f3bfa6562ecfa038f1ee728 ended up being live for quite a long time. Reference browser has now nightlies. I'm putting reference browser back.